cmd/cogito/main.go

```diff
@@ -37,16 +36,27 @@ func mainErr(in io.Reader, out io.Writer, logOut io.Writer, args []string) error
 	if err != nil {
 		return fmt.Errorf("reading stdin: %s", err)
 	}
+
 	logLevel, err := peekLogLevel(input)
 	if err != nil {
 		return err
 	}
-	log := hclog.New(&hclog.LoggerOptions{
-		Name:        "cogito",
-		Level:       hclog.LevelFromString(logLevel),
-		Output:      logOut,
-		DisableTime: true,
-	})
+	var level slog.Level
+	if err := level.UnmarshalText([]byte(logLevel)); err != nil {
+		return fmt.Errorf("%s. (valid: DEBUG, INFO, WARN, ERROR)", err)
```

Comment:
Ah.
I think I need to double-check this :-(

Reply:
Why?

You did the same in skyguide. It works fine.
https://github.com/Pix4D/skyguide/blob/28ac17a5f0409d622fec7ee1bd92b24dfdb76add/pkg/skyguide/app.go#L69

Reply:
I had a vague recollection we were doing something particular with the log levels. I now reconstructed the situation, see the second commit I pushed.

Do not be confused by the fact that I also made the strings lowercase, that is purely cosmetic to better match with the README, both cases are accepted. The real difference is not in the code but in the changes to the README and CHANGELOG.

Reply:
I see. Thanks. Good catch.